set.mm

```diff
@@ -66033,6 +66100,16 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
     ( wcel wa cxp wfn cop cfv wceq wbr wb opelxpi fnbrfvb sylan2 ) AEGBFGHDEFIZ
     JABKZSGTDLCMTCDNOABEFPSTCDQR $.
 
+  ${
+    $d A y $.  $d B y $.  $d F y $.  $d X y $.
+    $( There is exactly one codomain element for each element of the domain of
+       a function.  (Contributed by AV, 20-Apr-2025.) $)
+    fdmeu $p |- ( ( F : A --> B /\ X e. A ) -> E! y e. B ( F ` X ) = y ) $=
```

Comment:
antecedent not needed: `E! y e. B ( F ` X ) = y` without the antecedent can be proved from
https://us.metamath.org/mpeuni/eueq.html

Reply:
Actually I'm wrong: `E! y` is provable but not `E! y e. B`

Reply:
Yes, it is not that easy...